### PR TITLE
add NAT port forwarding notes

### PR DESCRIPTION
**Pull request by elmariofredo**
when virtual machine use NAT we need to do additional port forwarding steps

### CODE REVIEW

**Comment by zbwright** [comment]
@crawford is this true? if so, I can edit for language and approve.

**Comment by crawford** [comment]
@zbwright yes, this is accurate.

File: os/booting-on-virtualbox.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Click on 'OK' button and the virtual machine will be ready to be started.
 
 ## Logging in
 
-Networking can take a bit of time to come up under VirtualBox and you will need to know the IP in order to connect to it. Press enter a few times at the login prompt and you see an IP address pop up.
+Networking can take a bit of time to come up under VirtualBox and you will need to know the IP in order to connect to it. Press enter a few times at the login prompt and you see an IP address pop up. If you see VirtualBox NAT ip 10.0.2.15 you have go to virtual machine settings again then click on Network tab and then on Port Forwarding. In dialog add rule as follows Host Port: 2222; Guest Port 22 then you will be able to connect using command `ssh core@localhost -p2222`
```
Comment (zbwright) [inline os/booting-on-virtualbox.md]: hey - thank you, and slight edit, please:
Networking can take a bit of time to come up under VirtualBox and you will need to know the IP in order to connect to it. Press enter a few times at the login prompt to see an IP address pop up. If you see VirtualBox NAT IP 10.0.2.15, go to virtual machine settings and click the Network tab, then Port Forwarding. Add the rule "Host Port: 2222; Guest Port 22", then connect using the command `ssh core@localhost -p2222`.

**Comment by elmariofredo** [comment]
@zbwright thanks for wording it's updated now.

**Pull request merged**